index_algolia.js

```diff
@@ -2,8 +2,12 @@ const algoliasearch = require('algoliasearch');
 
 const data = require('./_posts/codelabs/index.json');
 
-const client = algoliasearch(process.env.APP_ID, process.env.ADMIN_API_KEY);
-const index = client.initIndex('codelabs');
+const LOCAL_STORAGE_KEY = process.env.NODE_ENV === 'production' ? 'codelabs' : 'codelabs';
```

Comment:
tu voulais pas plutôt ça ? :

```suggestion
const LOCAL_STORAGE_KEY = process.env.NODE_ENV === 'production' ? 'codelabs' : 'codelabs_dev';
```

Reply:
ah oui